frontend/src/concepts/pipelines/content/createRun/contentSections/TriggerTypeField.tsx

```diff
@@ -13,20 +13,24 @@ import {
   RunTypeScheduledData,
   ScheduledType,
 } from '~/concepts/pipelines/content/createRun/types';
-import NumberInputWrapper from '~/components/NumberInputWrapper';
-import { replaceNonNumericPartWithString, replaceNumericPartWithString } from '~/utilities/string';
 import {
   DEFAULT_CRON_STRING,
   DEFAULT_PERIODIC_OPTION,
 } from '~/concepts/pipelines/content/createRun/const';
+import NumberInputWrapper from '~/components/NumberInputWrapper';
+import { extractNumberAndTimeUnit } from './utils';
 
 type TriggerTypeFieldProps = {
   data: RunTypeScheduledData;
   onChange: (scheduledData: RunTypeScheduledData) => void;
 };
 
+const INPUT_MAX_VALUE = 9999999999;
```

Comment:
Alright, after testing with the current KF UI. We should not hard code the input value here because all the values will be finally transformed into seconds. When you choose a different unit, the input max limit shouldn't be the same. Let's revert this change and let the KF server throw the error.
cc @Gkrumbach07 let me know if you have a better solution because I don't know what the limit here should be. In KF UI when you input some value big (depends on the unit because all the values will be converted into seconds) it will throw the same error as well. We can leave this to KF server to handle. User should not input that big value for recurring runs.